addons/binding/org.openhab.binding.nest/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,45 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.binding.nest.internal
+Ignore-Package: org.openhab.binding.nest.internal
+Bundle-ManifestVersion: 2
+Bundle-Name: Nest Binding
+Bundle-SymbolicName: org.openhab.binding.nest;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Bundle-DocURL: http://www.openhab.org
+Import-Package: com.google.common.collect,
+ com.google.gson,
+ com.google.gson.annotations,
+ org.apache.commons.httpclient,
+ org.apache.commons.httpclient.auth,
+ org.apache.commons.httpclient.methods,
+ org.apache.commons.httpclient.params,
+ org.apache.commons.httpclient.util,
+ org.apache.commons.io,
+ org.apache.commons.lang,
+ org.apache.commons.lang.builder,
+ org.apache.commons.logging,
+ org.eclipse.jetty.http,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.eclipse.smarthome.io.net.http,
+ org.openhab.binding.nest,
+ org.openhab.binding.nest.handler,
+ org.osgi.framework,
+ org.slf4j
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.openhab.binding.nest,
+ org.openhab.binding.nest.handler,
+ org.openhab.binding.nest.config,
+ org.openhab.binding.nest.internal,
+ org.openhab.binding.nest.internal.data,
+ org.openhab.binding.nest.discovery
+Require-Bundle: org.eclipse.smarthome.config.discovery,
```

Comment:
According to the [Coding Guidelines](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles), the manifest must not contain any “Require-Bundle” entries. Instead, “Import-Package” must be used.